src/main/java/org/jabref/gui/preview/PreviewPanel.java

```diff
@@ -167,4 +154,20 @@ public void setEntry(BibEntry entry) {
     public void print() {
         previewView.print();
     }
+
+    public void nextPreviewStyle() {
+        cyclePreview(Globals.prefs.getPreviewPreferences().getPreviewCyclePosition() + 1);
```

Comment:
Please extract `Globals.prefs` as a constructor argument (probably replacing `PreviewPreferences`) as we want to get rid of `Globals` as much as possible.